requirements.txt

```diff
@@ -14,3 +14,8 @@ statsmodels==0.11.1
 tables==3.6.1
 umap-learn==0.4.6
 xarray==0.12.3
+Twisted~=20.3.0
+pyOpenSSL
+service_identity
```

Comment:
I wonder if simply including `kiosk-client` will do the trick. Looking at `setup.py` of the `kiosk-client` repo, it looks like it's configured to install `treq` on a `pip install`, meaning that if all goes well, we might be able to remove at least `twisted`, if not also `pyOpenSSL` and `service_identity` (assuming `treq` installs at least `twisted` as well).

Reply:
Yes it does! I only checked the requirements file itself (of kiosk-client), but as you and @ngreenwald said, kiosk-client installs all the necessary modules. thanks